main/src/library/Nel.flix

```diff
@@ -409,14 +409,14 @@ namespace Nel {
     /// Applies `g` to the initial element of `l` and combines it
     /// with the remainder of `l` using `f` going from right to left.
     ///
-    pub def reduceRightTo(f: (a, b) -> b & ef1, g: a -> b & ef2, l: Nel[a]): b & (ef1 and ef2) =
+    pub def reduceRightTo(f: (a, b) -> b & ef1 \ ef1_, g: a -> b & ef2 \ ef2_, l: Nel[a]): b & (ef1 and ef2) \ (ef1_ + ef2_) =
```

Comment:
Whats with the signature?

Reply:
And is a TODO needed?

Reply:
The continuation needs to use the effect part of the type, but that is not available when the type is implicutly split. so we have to add the explicit purity and effect separately in the signature.